### PR TITLE
Implement visual translation

### PR DESCRIPTION
**Pull request by soywod**
Translate and replace visually selected text

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -22,8 +22,11 @@ Make sure you've installed [translate-shell](https://github.com/soimort/translat
 Usage
 -----
 
+- `:'<,'>VTranslate en:fr`
```
Comment (VincentCordobes) [inline README.md]: Can we name this `:TranslateReplace` ? 
So we have:
- `:Translate`
- `:TranslateReplace`
and later on: 
- `:TranslateVisual`


And put it in 2nd position, so it's clear the base cmd is `:Translate`

File: doc/translate.txt [hunk 2]
```diff
@@ -23,7 +23,13 @@ Make sure you've installed {translate-shell}{2}
 ==============================================================================
 2. Usage                                                     *translate-usage*
 
-Translate source → target
+Translate source → target and replace the selection
+See https://github.com/soimort/translate-shell#code-list for code list
+>
+  :'<,'>VTranslate [source]:target
```
Comment (VincentCordobes) [inline doc/translate.txt]: same here in 2nd position ?

File: plugin/translate.vim [hunk 3]
```diff
@@ -8,26 +8,32 @@ augroup translate
...
-    call s:msg_error('translate-shell not found. Please install it.')
-    return
-  endif 
+  if !s:check_executable() | return | endif
 
   call s:translate_clear()
 
   silent! %y
   botright 8new
+  set buftype=nofile
```
Comment (VincentCordobes) [inline plugin/translate.vim]: stylé!!! c'est ça que je voulais ^^

File: README.md [hunk 4]
```diff
@@ -23,8 +23,10 @@ Usage
 -----
 
 - `:Translate en:fr`
-    - translate **english → french** (see [here](https://github.com/soimort/translate-shell#code-list) for code list)
+    - translate **english → french** (see [here](https://github.com/soimort/translate-shell#code-list) for code list) and put translation in a buffer
     
+- `:'<,'>TranslateReplace en:fr`
+    - translate **english → french** (see [here](https://github.com/soimort/translate-shell#code-list) for code list) and replace the selection
```
Comment (VincentCordobes) [inline README.md]: Can you remove the link to code list ? 😛 it's written just above

File: doc/translate.txt [hunk 5]
```diff
@@ -23,13 +23,20 @@ Make sure you've installed {translate-shell}{2}
...
-Translate source → target
+Translate source → target and put translation in a buffer
 See https://github.com/soimort/translate-shell#code-list for code list
 >
   :Translate [source]:target
 <
-    
+
+Translate source → target and replace the selection
+See https://github.com/soimort/translate-shell#code-list for code list
```
Comment (VincentCordobes) [inline doc/translate.txt]: link 🗑

Comment (soywod) [reply doc/translate.txt]: Done !

File: README.md (hunk 4 above)
Comment (soywod) [reply README.md]: Done !

File: README.md (hunk 1 above)
Comment (soywod) [reply README.md]: ✔️

File: doc/translate.txt (hunk 2 above)
Comment (soywod) [reply doc/translate.txt]: ✔️

**Comment by VincentCordobes** [comment]
Thanks you @soywod 🙂

**Pull request merged**